packages/ui/src/memberships/helpers.ts

```diff
@@ -0,0 +1,20 @@
+import { MemberRole } from './types'
+
+export function memberRoleAbbreviation(role: MemberRole) {
+  if (role.groupName === 'Member Role') {
+    return 'M'
+  }
+
+  return `${role.groupName.charAt(0).toUpperCase()}${role.isLeader ? 'L' : 'W'}`
+}
+
+export function memberRoleTitle(role: MemberRole) {
+  if (role.groupName === 'Member Role') {
+    return role.groupName
+  }
+
+  const position = role.isLeader ? 'Leader' : 'Worker'
+  const group = role.groupName
+
+  return `${group} ${position}`
+}
```

Comment:
Overall the roles are working pretty nice now :)

I would like to ask you for two things:

1.  Add tests (Let's have it defined how this work with tests)
2.  The role should be formatted as "Forum Worker". However, we have "forum" in the group name so we should have the role name format independent of the underlying data.

Reply:
@jodator, Just tests for this 2 functions?

Reply:
Yes, for these 2 functions.

Reply:
✅ Tests
✅ Capitalization